Sources/HummingbirdAuth/Authenticator/Authenticator.swift

```diff
@@ -16,7 +16,7 @@ import Hummingbird
 import NIOCore
 
 /// Protocol for objects that can be returned by an `HBAuthenticator`.
-public protocol HBAuthenticatable {}
+public protocol HBAuthenticatable: Sendable {}
```

Comment:
@Joannis Slightly concerned I'm going to have to back out this change given the last release of Vapor. I guess I won't be able to conform Fluent models to `HBAuthenticatable` if I require them to be Sendable

Reply:
going to merge and deal with this in a separate PR. I have an idea